scripts/makePlots_datacardsCPInitialState.py

```diff
@@ -686,9 +693,10 @@ def remove_procs_and_systs_with_zero_yield(proc):
 	
 	if args.n_plots[0] != 0 and "t2w" in args.steps:
 		# tools.parallelize(_call_command, hadd_commands, n_processes=args.n_processes)
-		datacards_module._call_command([
-			"$CMSSW_BASE/src/HiggsAnalysis/KITHiggsToTauTau/scripts/hadd_shapes.sh {OUTPUT_FOLDER}".format(OUTPUT_FOLDER=os.path.join(args.output_dir, "shapes", args.output_suffix))
-		])
+		for channel in ["em","et","mt","tt","ttbar"]:
```

Comment:
ttbar as a channel is misleading. Could you explain why you implement it like this?